cmd/mdmctl/mdmcert.go

```diff
@@ -169,6 +170,7 @@ func (cmd *mdmcertCommand) runPush(args []string) error {
 		flCN       = flagset.String("cn", "micromdm-user", "CommonName for the CSR Subject.")
 		flPKeyPass = flagset.String("password", "", "Password to encrypt/read the RSA key.")
 		flKeyPath  = flagset.String("private-key", filepath.Join(mdmcertdir, pushCertificatePrivateKeyFilename), "Path to the push certificate private key. A new RSA key will be created at this path.")
+		flNoLogging	 = flagset.Bool("nolog",false,"Does not log request to server so not configuration file required.")
```

Comment:
`log` is an odd choice for sending an RPC. `-local-only` or `-no-upload` would be more clear.